tests/framework/base/ModuleTest.php

```
@@ -36,214 +51,698 @@ public function testTrueParentModule(): void
+        self::assertSame(
+            __DIR__ . '/stub/module',
+            $module->getBasePath(),
+            'Default must be the class file directory.',
+        );
     }
 
-    public function actionTest2(): void
+    public function testSetViewPathResolvesAlias(): void
     {
-        ModuleTest::$actionRuns[] = $this->action->uniqueId;
+        Yii::setAlias('@moduleViews', __DIR__ . '/stub/views');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Yii' in method 'testSetViewPathResolvesAlias'. (undefined)

(StaticAccess)